extensions/configoperations/permutations/example_test.go

```diff
@@ -0,0 +1,37 @@
+package permutations
+
+import (
+	"fmt"
+
+	"github.com/rancher/shepherd/extensions/configoperations"
+)
+
+func ExamplePermute() {
```

Comment:
Previously resolved, but can you put a godoc comment? You have examples below, but would be good to have a godoc comment as well.

Reply:
Sorry, looks like I had this change but didn't merge it before. Fixed.